game/graphic/window.cc

```diff
@@ -312,7 +317,11 @@ void Window::render(Game &game, std::function<void (void)> drawFunc) {
 	updateStereo(stereo ? getSeparation() : 0.0f);
 	glerror.check("setup");
 	// Can we do direct to framebuffer rendering (no FBO)?
-	if (!stereo || type == 2) { view(stereo); drawFunc(); return; }
+	if (!stereo || type == 2) {
```

Comment:
Maybe nice to refactor `type` to an enumeration? so we have staticly easy to rename types?
If this is too hard because of the configuration not quite supporting enums then let's stick with this for now

Reply:
Tried it, looks better IMO